app/src/main/java/net/tensory/rxjavatalk/models/Combatant.java

```diff
@@ -8,6 +8,13 @@
     private int armySize;
     private int dragonCount;
 
+    public Combatant(House house, double debt, int armySize, int dragonCount) {
+        this.house = house;
+        this.debt = debt;
```

Comment:
I think we'll want to separate out debt from the Battles, but I think we can proceed with it as part of the `Compantant` class for this commit